sdk/tables/Azure.Data.Tables/src/autorest.md

```diff
@@ -8,9 +8,17 @@ Run `dotnet build /t:GenerateCode` to generate code.
 ``` yaml
 title: Azure.Data.Tables
 input-file:
-    - $(this-folder)/swagger/table.json
+    - https://github.com/Azure/azure-rest-api-specs/blob/bda39d9be69b9e838eb41e5b71964a567a627cbc/specification/cosmos-db/data-plane/Microsoft.Tables/preview/2019-02-02/table.json
 namespace: Azure.Data.Tables
 include-csproj: disable
 ```
 
-The direct swagger file reference is temporary until the following issue is addressed https://github.com/Azure/azure-sdk-for-net/issues/13559
+### Fix Response type for QueryEntitiesWithPartitionAndRowKey
+
+``` yaml
+directive:
```

Comment:
Is this tracked anywhere?

Reply:
Kind of - https://github.com/Azure/azure-rest-api-specs/pull/10194#issuecomment-662709937

I need to follow up on that PR though as it aged out over the break.

Reply:
I went ahead and opened a new PR for this change.
https://github.com/Azure/azure-rest-api-specs/pull/12280